ludwig/model_export/coreml_exporter.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+
+import coremltools as ct
+import torch
+from model import MyNet
+
+from ludwig.api import LudwigModel
+from ludwig.model_export.base_model_exporter import BaseModelExporter
+
+ class CoreMLExporter(BaseModelExporter):
+    def export_classifier(self, model_path, export_path):
+        ludwig_model = LudwigModel.load(model_path)
+        model = CoreMLExporter(ludwig_model.model)  # Wrap the model
```

Comment:
Is this `model` referenced anywhere? It is overwritten on line 21

Reply:
its used in this line of code:  traced_model = torch.jit.trace(model, example_input)